packages/atomic/src/components/commerce/atomic-commerce-breadbox/atomic-commerce-breadbox.new.stories.tsx

```diff
@@ -34,30 +55,29 @@ export const Default: Story = {
         <atomic-commerce-facets> </atomic-commerce-facets>
       </div>
     `,
+    decoratorSearch,
   ],
   play: async (context) => {
     await play(context);
     await playExecuteFirstSearch(context);
-    const {canvasElement, step} = context;
-    const canvas = within(canvasElement);
-    await step('Wait for the facet values to render', async () => {
-      await waitFor(
-        () => expect(canvas.getByShadowTitle('People')).toBeInTheDocument(),
-        {
-          timeout: 30e3,
-        }
-      );
-    });
-    await step('Select a facet value', async () => {
-      const facet = canvas.getByShadowTitle('People');
-      await userEvent.click(facet);
-      await waitFor(
-        () =>
-          expect(
-            canvas.getByShadowTitle('Object type: People')
-          ).toBeInTheDocument(),
-        {timeout: 30e3}
-      );
-    });
+  },
+};
+
+export const WithDateFacet: Story = {
```

Comment:
Instead of having two stories, could we create a product listing configuration that would contain all four types of facets and use that as the default?

https://sports-dev.barca.group/browse/promotions/surf-with-us-this-year is only missing a category facet. We could create / use a listing configuration that has them all and somehow indicate in its name that it's for our tests.

Reply:
I guess we could do that yes!